santoku/aws/secrets_manager_handler.py

```diff
@@ -23,7 +24,7 @@ class SecretsManagerHandler:
 
     """
 
-    def __init__(self):
+    def __init__(self, **kwargs):
```

Comment:
What's the purpose of adding kwargs?

Reply:
To provide the possibility to instantiate the class specifying the AWS credentials. Sometimes it might be necessary to have two handlers, one for each account or one for each region for example.